projects/testing-library/src/lib/testing-library.ts

```diff
@@ -42,6 +42,14 @@ type SubscribedOutput<T> = readonly [key: keyof T, callback: (v: any) => void, s
 const mountedFixtures = new Set<ComponentFixture<any>>();
 const safeInject = TestBed.inject || TestBed.get;
 
+export function getInstanceWithProviders<T>(serviceClass: Type<T>, providers: any[]): T {
```

Comment:
Hello, thanks for raising the PR.

Personally, I don't see a lot of value of creating this function.
Having this function in your own codebase is totally fine, and also allows more flexibility to configure the testbed.

What would the main reason be in your opinion to have this available in Angular Testing Library?